frontend/src/component/admin/instance-privacy/index.tsx

```diff
@@ -0,0 +1,151 @@
+import AccessContext from 'contexts/AccessContext';
+import React, { useContext, useState } from 'react';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { PageHeader } from 'component/common/PageHeader/PageHeader';
+import { Box, styled } from '@mui/material';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { InstancePrivacy } from './InstancePrivacy';
+import { useTelemetry } from 'hooks/api/getters/useTelemetry/useTelemetry';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IFeatureActivenessManagementInfo {
+    enabled: IActivenessManagementInfo;
+    disabled: IActivenessManagementInfo;
+}
+
+interface IActivenessManagementInfo {
+    environmentVariables: String[];
+    changeInfoText: String;
+}
+
+const StyledBox = styled(Box)(({ theme }) => ({
+    display: 'grid',
+    gap: theme.spacing(4),
+}));
+
+const versionCollectionDetails = {
+    title: 'Version data collection',
+    infoText:
+        "We collect the version of Unleash that you're using. We use this information to inform your Unleash instance of latest updates and and critical security patches.",
+
+    concreteDetails: {
+        'Instance ID': 'A unique ID generated for your instance',
+        Version: "The version of Unleash that you're using",
+    },
+};
+
+const featureCollectionDetails = {
+    title: 'Feature data collection',
+    infoText:
+        'We collect data about your instance to improve the Unleash product user experience. Also we use the data in case you need help from our support. Data collection is for internal use and is not shared with third parties outside Unleash. As we want you to be in control of your data, we will leave it up to you to allow us to collect your data.',
```

Comment:
Just some super minor tweak suggestions. Up to you ☺️ 

```suggestion
        'We collect data about your instance to improve the Unleash product user experience. We may also use the data in case you need help from our support team. Data collection is for internal use only and is not shared with third parties outside Unleash. As we want you to be in control of your data, we will leave it up to you to allow us to collect your data.',
```